roles/libvirt_manager/tasks/deploy_layout.yml

```diff
@@ -169,12 +169,28 @@
   vars:
     vm_type: "{{ _vm.value }}"
     vm: "{{ _vm.key }}"
+    _matcher: >-
+      {{
+        '(cifmw-)?' ~
+        '([a-z0-9]+)-[0-9]+'
+      }}
+    _ocp_master_matcher: >-
+      {{
+        '^ocp-(\d+)$'
+      }}
     vm_data: >-
       {{
         _cifmw_libvirt_manager_layout.vms[vm_type]
       }}
     pub_key: "{{ pub_ssh_key.content | b64decode }}"
     priv_key: "{{ priv_ssh_key.content | b64decode }}"
+    # If $vm is an OCP master node, capture it's index
+    # and craft the _vm_img varible, examples:
+    # ocp-0 -> ocp_master-0.qcow (match and modify)
```

Comment:
`ocp-(master|worker)-ID.qcow2` actually, with the last change converging all names to use hyphens instead of underscores.

Reply:
Removed